### ISSUE TITLE
[bug] help running bactopia

### ISSUE THREAD

**Issue by telatin** [bactopia/bactopia#203]
**Description**
Pipeline exits prematurely without error,s, in the `logs/fastq_status/fastq_status.err` I find:

```
  File "/home/ubuntu/volume/miniconda3/envs/bactopia/share/bactopia-1.6.x/bin/check-fastqs.py", line 31
    error_msg = (f"{sample} FASTQ(s) contain {total_reads} total reads. This does not \n"
                                                                                        ^
SyntaxError: invalid syntax
```

**Steps to Reproduce**

```
mamba create -n bactopia -c conda-forge -c bioconda bactopia
conda activate bactopia
bactopia datasets --cpus 8 --species 'Klebsiella pneumoniae' --include_genus --outdir kpn
bactopia prepare 704/ > 704.fastqs
bactopia --fastqs 704.fastqs --datasets kpn/ --cpus 8 --outdir bactopia_704 --species "Klebsiella pneumoniae"
```

**Expected Behavior**
I'm trying bactopia for the first time (but I already love it)

**Execution Environment**
Using a VM with Ubuntu (64 cores, 80 Gb RAM), I installed bactopia as follows

- Bactopia Version: **bactopia 1.6.1**
- OS: Ubuntu 16.04.5 LTS
- Environment: Ubuntu with conda (4.9.2) /mamba (0.8.2)
  - Python inside bactopia env: Python 3.8.8

**Additional Information**
Any other information you think might be helpful in squashing this bug.

**Comment by rpetit3** [comment]
Hi @telatin! Thanks for trying out Bactopia! I'm going to see if I can reproduce this one, and get it fixed.

In the meantime, if don't think there are any issues with your input FASTQs, you can can use `--skip_fastq_check`. It's mostly a sanity check and helpful dealing with ENA/SRA data.

**Comment by rpetit3** [comment]
Good news, this should be fixed https://github.com/bactopia/bactopia/commit/1da277b8c373338e1acd666bea4955e91ddf044c

It will be the next version (v1.6.2)

**Comment by rpetit3** [comment]
I've released v1.6.2, which should fix this issue. Please feel free to reopen if it doesn't! Thank you for your help!
